artifacts/examples/generic-attestation-policy-example.yaml

```diff
@@ -4,5 +4,4 @@ metadata:
   name: my-gap
   namespace: default
 spec:
-  attestationAuthorityNames:
-  - kritis-authority
+  attestationAuthorityName: kritis-authority
```

Comment:
This file is an example of a GAP. From reading the comments in the PR, this should only be changing ISPs. Perhaps this is a mistake?

Reply:
Yes, thanks for pointing out! Will fix.